src/MoBi.Core/Commands/AddTopContainerCommand.cs

```diff
@@ -1,24 +1,19 @@
-﻿using System.Linq;
-using MoBi.Core.Domain.Model;
-using MoBi.Core.Events;
+﻿using MoBi.Core.Domain.Model;
 using OSPSuite.Core.Commands.Core;
 using OSPSuite.Core.Domain;
-using OSPSuite.Core.Domain.Services;
-using OSPSuite.Utility.Collections;
-using OSPSuite.Utility.Extensions;
 
 namespace MoBi.Core.Commands
 {
    public class AddTopContainerCommand : AddObjectBaseCommand<IContainer, MoBiSpatialStructure>
    {
-      public AddTopContainerCommand(MoBiSpatialStructure parent, IContainer itemToAdd) : base(parent, itemToAdd, parent)
```

Comment:
I HATE IT when we are so lazy to rename the dynamic variable based on their specific type. Generic does not mean lazy